libs/vue-route-guard/project.json

```diff
@@ -4,6 +4,20 @@
   "sourceRoot": "libs/vue-route-guard/src",
   "targets": {
     "build": {
+      "executor": "@nrwl/workspace:run-commands",
+      "options": {
+        "parallel": false,
+        "commands": [
+          {
+            "command": "nx build-app 'vue-route-guard'"
+          },
+          {
+            "command": "node ./libs/vue-route-guard/buildPackage.ts"
```

Comment:
**suggestion**: We could simplify the code by running `npx cpy from-path to-path`